configs/HPECray-zen3-MI250X-Slingshot/variables.yaml

```diff
@@ -1,27 +0,0 @@
-# Copyright 2023 Lawrence Livermore National Security, LLC and other
-# Benchpark Project Developers. See the top-level COPYRIGHT file for details.
-#
-# SPDX-License-Identifier: Apache-2.0
-
-variables:
-  gtl_flag: ''  # to be overwritten by tests that need GTL
-  rocm_arch: 'gfx90a'
-  scheduler: 'flux'
-  flux_batch_time: '120m'
-  slurm_batch_time: '02:00'
-  batch_time: '{{scheduler}_batch_time}'
-  flux_mpi_command: 'flux run -N {n_nodes} -n {n_ranks}'
-  flux_batch_submit: 'flux batch {execute_experiment}'
-  flux_batch_nodes: '# flux: -N {n_nodes}'
-  flux_batch_ranks: '# flux: -n {n_ranks}'
-  flux_batch_timeout: '# flux: -t {batch_time}'
-  slurm_mpi_command: 'srun -N {n_nodes} -n {n_ranks}'
-  slurm_batch_submit: 'sbatch {execute_experiment}'
-  slurm_batch_nodes: '#SBATCH -N {n_nodes}'
-  slurm_batch_ranks: '#SBATCH -n {n_ranks}'
-  slurm_batch_timeout: '#SBATCH -t {batch_time}:00'
-  mpi_command: '{{scheduler}_mpi_command}'
-  batch_submit: '{{scheduler}_batch_submit}'
-  batch_nodes: '{{scheduler}_batch_nodes}'
-  batch_ranks: '{{scheduler}_batch_ranks}'
-  batch_timeout: '{{scheduler}_batch_timeout}'
```

Comment:
Is the intent to remove all of `HPECray-zen3-MI250X-Slingshot`? I get the impression it should, since now there is `generic-HPECray-zen3-MI250X-Slingshot`. If any of it stays, then I think this should stay (or a modified form of it that narrows things down to the scheduler.

(overall though it looks like the `git mv` was not complete for HPECray-zen3-MI250X-Slingshot, since several files in this diff are displayed as entirely new for generic-HPECray-zen3-MI250X-Slingshot)

Reply:
I moved the HPECray system to LLNL-Tioga, since I kept that identical (except switching the scheduler to be Flux).  Then I made a (partial) copy for generic-HPECray, excluding site specific packages (spack externals), and using Slurm as the scheduler.

Did I miss anything?

Reply:
Sorry it's not clear IMO from the diff, but I thought there were remaining files in `HPECray-zen3-MI250X-Slingshot`. I tried pulling this PR and examining it locally, and the organization looks fine, so I'll mark this as resolved (I still think we should not use the term "generic" as part of the name though).